generators/client/files-angular.js

```diff
@@ -37,6 +37,8 @@ const files = {
                 'tsconfig.json',
                 'tsconfig-aot.json',
                 'tslint.json',
```

Comment:
Why are you keeping `tslint.json`?

Reply:
`codelyzer` rules still rely on the `tslint`. There is an open ticket to migrate to `eslint` https://github.com/mgechev/codelyzer/issues/763.